docs/src/Popover.doc.js

```diff
@@ -571,6 +588,12 @@ To assist screen readers, we recommend passing the following ARIA attributes to
 
 - \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover attached to the anchor element. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
 - \`accessibilityExpanded\`: informs the screen reader whether Popover is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityControls\`: match with the \`id\` of the associated Popover whose contents or visibility are controlled by the interactive component so that screen reader users can identify the relationship between elements. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+
+For role attribute, use:
+- 'dialog' if the Popover is a dialog that prompts the user to enter information or require a response.
+- 'menu' if the Popover list of choices to the user.
```

Comment:
slight typo: - 'menu' if the Popover presents a list of choices to the user.

Reply:
DONE